### DISCUSSION TITLE
Win 10 Performance - Processor scheduling setting

### DISCUSSION THREAD

**Discussion by leoinlios** [jamulussoftware/jamulus#2074] [General]
Hi, Windows 10 - System Properties - Advanced Tab - Performance settings - Processor scheduling

There are 2 options. Adjust for best performance of **Programs** OR **Background services**. 

I read that for music production one must disable all Background apps, and must select **Background services** in the above setting. 

In the context of running the Jamulus client, i.e. not a server, just playing your instrument in Jamulus, what should be selected? 

I am trying to look for all the necessary Win 10 tweaks that must be done to improve my 10ms ping -> 30ms delay situation. 
(I appreciate that tweaking the OS may not reduce the delay, but I thought to start here)

Thank you

**Comment by gilgongo** [comment]
Not heard of people adjusting that, but it would be interesting to know if it made a difference. 

One way of telling might be to set up a server on your local machine, connect a client running on that machine to it, and see if there's less overall delay when you set the OS for background compared to programs (this is known as a "loopback test").

**Comment by pcar75** [comment]
I would be interested to know if your have worst Overal Dealy and sound quality problem  when using a server on the same machine [client in loopback connexion (127.0.0.1) to server]  Versus a Public nearby server.
 See #1855 for further discussion.
Thanks.

**Comment by leoinlios** [comment]
Thanks, will do the loopback test.

**Comment by Andrew999999** [comment]
I did a lot of testing to optimise many aspects of Win10 when I first started using Jamulus last year.  Eventually I gave up on the client side as nothing made a great deal of difference.  Mainly now I just make sure I'm not running backups or other heavy workloads at the same time.  However I use a number of private Windows Jamulus servers that I set up on AWS.  There I do prioritise the Jamulus process as realtime and it does help a little in reducing jitter.  You can do this from the command line when you invoke the server using something like this...  

start "" /realtime /d "C:\Program Files (x86)\Jamulus" /b "C:\Program Files (x86)\Jamulus\jamulus.exe"
